template/nuxt/nuxt.config.js

```diff
@@ -35,15 +35,19 @@ module.exports = {
   /*
   ** Global CSS
   */
-  css: [<% if (ui === 'element-ui') { %>'element-ui/lib/theme-chalk/index.css'<% }
-           else if (ui === 'tailwind') { %>'~/assets/css/tailwind.css'<% }
-           else if (ui === 'vuetify') { %>'~/assets/style/app.styl'<% } %>],
+  css: [<% if (ui === 'element-ui') { %>'element-ui/lib/theme-chalk/index.css'<% } 
```

Comment:
If `]` is in a new line, the content after `[` is better also in new line and has correct indentation.